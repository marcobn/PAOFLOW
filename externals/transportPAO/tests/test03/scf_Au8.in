Infinite Gold Chain
8 atom/cell, Lattice Parameter = 2.5 Ang
   &CONTROL
      calculation = 'scf',
      restart_mode='from_scratch',
      verbosity='default',
      prefix='gold8',
      nstep=100,
      tstress = .false.,
      tprnfor = .true.,
      pseudo_dir = '../pseudo',
      outdir = './SCRATCH/'
   /
   &SYSTEM
      ibrav=8, 
      celldm(1)=20.0, 
      celldm(2)=1.0,
      celldm(3)=1.88972687777435527243
      nat=8, 
      ntyp=1,
      ecutwfc=50.0,
      ecutrho=200.0,
      occupations='smearing',
      smearing='gaussian',
      degauss=0.01,
   /
   &ELECTRONS
      electron_maxstep = 100,
      mixing_mode = 'plain',
      mixing_beta = 0.3,
      conv_thr =  1.0d-8,
   /
ATOMIC_SPECIES
  Au   196.9665   Au11pw91.mt.UPF
ATOMIC_POSITIONS {angstrom}
Au   0.0  0.0 0.0
Au   0.0  0.0 2.5
Au   0.0  0.0 5.0
Au   0.0  0.0 7.5
Au   0.0  0.0 10.0
Au   0.0  0.0 12.5
Au   0.0  0.0 15.0
Au   0.0  0.0 17.5
K_POINTS automatic
1 1 4 0 0 0

