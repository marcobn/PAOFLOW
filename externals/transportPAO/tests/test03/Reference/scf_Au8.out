
     Program PWSCF v.5.0.2 (svn rev. 9392) starts on 17Apr2014 at 20:21:39 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote.php

     Parallel version (MPI), running on     6 processors
     R & G space division:  proc/nbgrp/npool/nimage =       6

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3
     Reading input from /home/ferretti/want/tests/test16/scf_Au8.in

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

     Found symmetry operation: I + (  0.0000  0.0000  0.1250)
     This is a supercell, fractional translations are disabled
 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min        1061    1061    269               120331   120331   15567
     Max        1062    1062    270               120334   120334   15574
     Sum        6369    6369   1617               721995   721995   93423
 


     bravais-lattice index     =            8
     lattice parameter (alat)  =      20.0000  a.u.
     unit-cell volume          =   15117.8150 (a.u.)^3
     number of atoms/cell      =            8
     number of atomic types    =            1
     number of electrons       =        88.00
     number of Kohn-Sham states=           53
     kinetic-energy cutoff     =      50.0000  Ry
     charge density cutoff     =     200.0000  Ry
     convergence threshold     =      1.0E-08
     mixing beta               =       0.3000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      =  SLA  PW   GGX  GGC ( 1 4 2 2 0)
     EXX-fraction              =        0.00

     celldm(1)=  20.000000  celldm(2)=   1.000000  celldm(3)=   1.889727
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   1.000000   0.000000 )  
               a(3) = (   0.000000   0.000000   1.889727 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.000000 )  
               b(3) = (  0.000000  0.000000  0.529177 )  


     PseudoPot. # 1 for Au read from file:
     ../pseudo/Au11pw91.mt.UPF
     MD5 check sum: e270e13d0e45fa10676aee720c7982d9
     Pseudo is Norm-conserving + core correction, Zval = 11.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  331 points,  2 beta functions with: 
                l(1) =   1
                l(2) =   2

     atomic species   valence    mass     pseudopotential
        Au            11.00   196.96650     Au( 1.00)

     16 Sym. Ops., with inversion, found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Au  tau(   1) = (   0.0000000   0.0000000   0.0000000  )
         2           Au  tau(   2) = (   0.0000000   0.0000000   0.2362158  )
         3           Au  tau(   3) = (   0.0000000   0.0000000   0.4724315  )
         4           Au  tau(   4) = (   0.0000000   0.0000000   0.7086473  )
         5           Au  tau(   5) = (   0.0000000   0.0000000   0.9448631  )
         6           Au  tau(   6) = (   0.0000000   0.0000000   1.1810788  )
         7           Au  tau(   7) = (   0.0000000   0.0000000   1.4172946  )
         8           Au  tau(   8) = (   0.0000000   0.0000000   1.6535104  )

     number of k points=     3  gaussian smearing, width (Ry)=  0.0100
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.5000000
        k(    2) = (   0.0000000   0.0000000   0.1322943), wk =   1.0000000
        k(    3) = (   0.0000000   0.0000000  -0.2645885), wk =   0.5000000

     Dense  grid:   721995 G-vectors     FFT dimensions: (  96,  96, 180)

     Largest allocated arrays     est. size (Mb)     dimensions
        Kohn-Sham Wavefunctions        12.17 Mb     (  15054,   53)
        NL pseudopotentials            14.70 Mb     (  15054,   64)
        Each V/rho on FFT grid          4.22 Mb     ( 276480)
        Each G-vector array             0.92 Mb     ( 120334)
        G-vector shells                 0.23 Mb     (  30306)
     Largest temporary arrays     est. size (Mb)     dimensions
        Auxiliary wavefunctions        48.70 Mb     (  15054,  212)
        Each subspace H/S matrix        0.69 Mb     ( 212, 212)
        Each <psi_i|beta_j> matrix      0.05 Mb     (     64,   53)
        Arrays for rho mixing          33.75 Mb     ( 276480,   8)

     Check: negative/imaginary core charge=   -0.000021    0.000000

     Initial potential from superposition of free atoms
     Check: negative starting charge=   -0.045676

     starting charge   87.99942, renormalised to   88.00000

     negative rho (up, down):  0.457E-01 0.000E+00
     Starting wfc are   72 randomized atomic wfcs

     total cpu time spent up to now is       11.1 secs

     per-process dynamical memory:    90.0 Mb

     Self-consistent Calculation

     iteration #  1     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  3.7

     Threshold (ethr) on eigenvalues was too large:
     Diagonalizing with lowered threshold

     Davidson diagonalization with overlap
     ethr =  5.12E-04,  avg # of iterations =  2.0

     negative rho (up, down):  0.284E-01 0.000E+00

     total cpu time spent up to now is       49.2 secs

     total energy              =    -728.69726543 Ry
     Harris-Foulkes estimate   =    -728.89830937 Ry
     estimated scf accuracy    <       0.47975790 Ry

     iteration #  2     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  5.45E-04,  avg # of iterations =  2.0

     negative rho (up, down):  0.188E-01 0.000E+00

     total cpu time spent up to now is       67.5 secs

     total energy              =    -728.77990215 Ry
     Harris-Foulkes estimate   =    -728.78436622 Ry
     estimated scf accuracy    <       0.00907218 Ry

     iteration #  3     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.03E-05,  avg # of iterations =  5.3

     negative rho (up, down):  0.161E-02 0.000E+00

     total cpu time spent up to now is       90.8 secs

     total energy              =    -728.77479191 Ry
     Harris-Foulkes estimate   =    -728.78163492 Ry
     estimated scf accuracy    <       0.00392989 Ry

     iteration #  4     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  4.47E-06,  avg # of iterations =  3.0

     negative rho (up, down):  0.206E-02 0.000E+00

     total cpu time spent up to now is      113.1 secs

     total energy              =    -728.77568098 Ry
     Harris-Foulkes estimate   =    -728.77589921 Ry
     estimated scf accuracy    <       0.00074705 Ry

     iteration #  5     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  8.49E-07,  avg # of iterations =  2.7

     negative rho (up, down):  0.174E-02 0.000E+00

     total cpu time spent up to now is      131.8 secs

     total energy              =    -728.77572447 Ry
     Harris-Foulkes estimate   =    -728.77577256 Ry
     estimated scf accuracy    <       0.00013239 Ry

     iteration #  6     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.50E-07,  avg # of iterations =  2.0

     negative rho (up, down):  0.121E-03 0.000E+00

     total cpu time spent up to now is      149.7 secs

     total energy              =    -728.77560639 Ry
     Harris-Foulkes estimate   =    -728.77574669 Ry
     estimated scf accuracy    <       0.00015836 Ry

     iteration #  7     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.50E-07,  avg # of iterations =  2.0

     negative rho (up, down):  0.848E-04 0.000E+00

     total cpu time spent up to now is      167.8 secs

     total energy              =    -728.77561807 Ry
     Harris-Foulkes estimate   =    -728.77562317 Ry
     estimated scf accuracy    <       0.00001412 Ry

     iteration #  8     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.60E-08,  avg # of iterations =  1.7

     negative rho (up, down):  0.632E-04 0.000E+00

     total cpu time spent up to now is      185.5 secs

     total energy              =    -728.77561684 Ry
     Harris-Foulkes estimate   =    -728.77561919 Ry
     estimated scf accuracy    <       0.00000375 Ry

     iteration #  9     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  4.26E-09,  avg # of iterations =  2.0

     negative rho (up, down):  0.124E-05 0.000E+00

     total cpu time spent up to now is      203.7 secs

     total energy              =    -728.77561036 Ry
     Harris-Foulkes estimate   =    -728.77561738 Ry
     estimated scf accuracy    <       0.00000065 Ry

     iteration # 10     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  7.37E-10,  avg # of iterations =  2.0

     negative rho (up, down):  0.992E-07 0.000E+00

     total cpu time spent up to now is      224.3 secs

     total energy              =    -728.77560939 Ry
     Harris-Foulkes estimate   =    -728.77561046 Ry
     estimated scf accuracy    <       0.00000009 Ry

     iteration # 11     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.00E-10,  avg # of iterations =  2.3

     total cpu time spent up to now is      246.2 secs

     End of self-consistent calculation

          k = 0.0000 0.0000 0.0000 ( 90209 PWs)   bands (ev):

   -10.8373 -10.0346 -10.0346  -8.5084  -8.5084  -8.1544  -8.1544  -8.1544
    -8.1544  -7.8488  -7.8484  -7.1824  -7.1824  -7.1823  -7.1823  -6.7949
    -6.7948  -6.7010  -6.7010  -6.7009  -6.7009  -6.4674  -6.4673  -6.4673
    -6.4671  -6.4468  -6.2233  -6.2233  -6.2231  -6.2231  -6.2145  -6.2145
    -6.1189  -6.1187  -5.9208  -5.9208  -5.9208  -5.9208  -5.8142  -5.8141
    -5.4874  -5.4871  -5.3318  -5.2092  -5.2092  -3.3535  -3.3535  -1.1744
    -1.1744  -0.4746  -0.4746  -0.4746  -0.4746

          k = 0.0000 0.0000 0.1323 ( 90361 PWs)   bands (ev):

   -10.7863 -10.3823  -9.5964  -8.4861  -8.4859  -8.4859  -8.3076  -8.3076
    -7.9616  -7.9616  -7.4708  -7.4708  -7.2023  -6.8735  -6.8735  -6.7888
    -6.7887  -6.7411  -6.7410  -6.6518  -6.6516  -6.5328  -6.5327  -6.4322
    -6.4011  -6.4010  -6.3159  -6.2765  -6.2764  -6.2305  -6.2305  -6.1792
    -6.1790  -6.1437  -6.1258  -6.1256  -6.0843  -5.7285  -5.6412  -5.6412
    -5.5907  -5.3595  -5.2625  -5.2625  -5.1736  -4.1293  -2.4683  -1.1237
    -1.1237  -0.7564  -0.7564  -0.6928  -0.1470

          k = 0.0000 0.0000-0.2646 ( 90294 PWs)   bands (ev):

   -10.6340 -10.6339  -9.0760  -9.0759  -8.4186  -8.4186  -8.4186  -8.4186
    -7.7324  -7.7324  -7.7324  -7.7324  -6.7707  -6.7706  -6.7706  -6.7706
    -6.6114  -6.6114  -6.5950  -6.5950  -6.5949  -6.5949  -6.5526  -6.5526
    -6.5526  -6.5526  -6.3885  -6.3884  -6.3367  -6.3366  -6.3365  -6.3365
    -6.1463  -6.1462  -6.1461  -6.1460  -5.9237  -5.9237  -5.4440  -5.4439
    -5.4136  -5.4136  -5.4135  -5.4135  -4.7379  -4.7379  -1.5438  -1.5438
    -0.9787  -0.9787  -0.9787  -0.9787   0.2178

     the Fermi energy is    -5.1876 ev

!    total energy              =    -728.77560968 Ry
     Harris-Foulkes estimate   =    -728.77560942 Ry
     estimated scf accuracy    <          3.9E-09 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =    -950.02724775 Ry
     hartree contribution      =     520.28488978 Ry
     xc contribution           =    -305.80201640 Ry
     ewald contribution        =       6.77996698 Ry
     smearing contrib. (-TS)   =      -0.01120230 Ry

     convergence has been achieved in  11 iterations

     Forces acting on atoms (Ry/au):

     atom    1 type  1   force =     0.00000000    0.00000000    0.00000000
     atom    2 type  1   force =     0.00000000    0.00000000    0.00000101
     atom    3 type  1   force =     0.00000000    0.00000000    0.00000159
     atom    4 type  1   force =     0.00000000    0.00000000   -0.00000096
     atom    5 type  1   force =     0.00000000    0.00000000    0.00000000
     atom    6 type  1   force =     0.00000000    0.00000000    0.00000096
     atom    7 type  1   force =     0.00000000    0.00000000   -0.00000159
     atom    8 type  1   force =     0.00000000    0.00000000   -0.00000101

     Total force =     0.000003     Total SCF correction =     0.000026
     SCF correction compared to forces is large: reduce conv_thr to get better values

     Writing output data file gold8.save
 
     init_run     :     10.72s CPU     10.91s WALL (       1 calls)
     electrons    :    230.43s CPU    235.01s WALL (       1 calls)
     forces       :      2.21s CPU      2.25s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      8.92s CPU      9.05s WALL (       1 calls)
     potinit      :      0.80s CPU      0.83s WALL (       1 calls)

     Called by electrons:
     c_bands      :    185.34s CPU    189.18s WALL (      12 calls)
     sum_band     :     37.18s CPU     37.64s WALL (      12 calls)
     v_of_rho     :      5.87s CPU      5.93s WALL (      12 calls)
     mix_rho      :      1.22s CPU      1.24s WALL (      12 calls)

     Called by c_bands:
     init_us_2    :      1.03s CPU      1.16s WALL (      78 calls)
     cegterg      :    182.30s CPU    184.91s WALL (      36 calls)

     Called by *egterg:
     h_psi        :    161.35s CPU    161.94s WALL (     131 calls)
     g_psi        :      1.05s CPU      1.06s WALL (      92 calls)
     cdiaghg      :      1.24s CPU      1.27s WALL (     125 calls)

     Called by h_psi:
     add_vuspsi   :      5.23s CPU      5.26s WALL (     131 calls)

     General routines
     calbec       :      5.62s CPU      5.61s WALL (     134 calls)
     fft          :      4.30s CPU      4.27s WALL (     144 calls)
     fftw         :    155.70s CPU    156.18s WALL (   11050 calls)
     davcio       :      0.01s CPU      1.66s WALL (     114 calls)
 
     Parallel routines
     fft_scatter  :     57.90s CPU     58.20s WALL (   11194 calls)
 
     PWSCF        :  4m 3.76s CPU     4m 8.63s WALL

 
   This run was terminated on:  20:25:47  17Apr2014            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
